app/build.gradle.kts

```diff
@@ -29,6 +37,7 @@ android {
   defaultConfig {
     applicationId = "dev.msfjarvis.aps"
     testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    vectorDrawables { useSupportLibrary = true }
```

Comment:
Is this necessary?

Reply:
The Android Studio adds it automatically whenever you set up compose through it so I decided to keep it in.